packages/cli/src/executions/executions.service.ts

```diff
@@ -473,7 +468,7 @@ export class ExecutionsService {
 
 			if (!executions.length) return;
 
-			const idsToDelete = executions.map(({ id }) => id.toString());
+			const idsToDelete = executions.map(({ id }) => id);
```

Comment:
Unrelated to this PR, but looks like we only need the IDs rather than the entire entities.

Reply:
done. here and in a few other places.